staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -82,6 +82,7 @@ type StartFunction func(ctx context.Context, hashValue uint64) (execute bool, af
 type RequestDigest struct {
 	RequestInfo *request.RequestInfo
 	User        user.Info
+	Width       float64
```

Comment:
I thought we were planning on this being an `int`.  I think `uint` would be a reasonable choice too.

Reply:
I agree with Mike - let's not overgeneralize here.
If at some point we will realize we need float (we don't need now), we will change it - it's not external API, so it's easy to change.

Reply:
i have picked `unit`, yes, i agree that we will not need "width" to have fractional value in foreseeable future